data/json/recipes/recipe_food.json

```diff
@@ -4967,24 +4960,16 @@
   },
   {
     "type": "recipe",
-    "activity_level": "LIGHT_EXERCISE",
+    "activity_level": "NO_EXERCISE",
```

Comment:
same thing, this could be 2 recipes

Reply:
I look forward to the day in the future when we can remove all wheat-free recipes.
I think we can do this by simply adding a flag like 'CANNIBALISM'.

Maybe it's up to me to try to deal with that future - in a few months when I have nothing else to do.